CONTRIBUTING.md

```diff
@@ -158,7 +158,7 @@ We definitely appreciate pull requests that highlight or reproduce a problem, ev
         DW_BASE_URL=https://api.data.world/v0
         DW_CLIENT_ID= (client id from the DW Oauth Client created earlier in step 4.)
         DW_CLIENT_SECRET= (client secret from the DW Oauth Client created earlier in step 4.)
-        DW_AUTH_BASE_URL=https://data.world/oauth/authorize
+        DW_AUTH_BASE_URL=https://auth.data.world/oauth/authorize
```

Comment:
I don't think this is going to change anything :) where are the `.env` files? We probably need to update those.

Also worth mentioning, deploying this (via Heroku) is going to be blocked on Nikhil's PR!

Reply:
Good point, we should be changing it in the .env files, but is that something that each individual user would have to do when they are deploying slack apps? I don't think .env files are files that we commit to the repo, or I haven't seen any that have been committed. From[ CONTRIBUTING.md](https://github.com/datadotworld/slack-app/blob/main/CONTRIBUTING.md#5-add-env-variables), it seems like the user just has to create the .env file and update it by themselves.

Reply:
yes. This is on Heroku and Nikhil reminded me that the env vars are set there. I'll update the setting in Heroku when I deploy it.